plugins/UsersManager/vue/src/UserSecurity/UserSecurity.vue

```diff
@@ -109,6 +110,9 @@
           <td>
             {{ theToken.last_used ? theToken.last_used : translate('General_Never') }}
           </td>
+          <td>
+            {{ theToken.post_only ? translate('General_Yes') : translate('General_No') }}
```

Comment:
This compare might fail. When using MYSQLI `post_only` might be of type `string`. And a string `"0"` evaluates to `true`
```suggestion
            {{ parseInt(theToken.post_only, 10) === 1 ? 
               translate('General_Yes') : translate('General_No') }}
```